### PR TITLE
Support single gpu systems

### PR DESCRIPTION
**Pull request by white238**
Didn't realize this was zero-based indexed...

### CODE REVIEW

File: src/axom/spin/tests/CMakeLists.txt [hunk 1]
```diff
@@ -72,7 +72,7 @@ if ( RAJA_FOUND AND UMPIRE_FOUND )
     #   C++ exception with description "CUDAassert" thrown in the test body.
     set_property(TEST spin_bvh
                  APPEND
-                 PROPERTY ENVIRONMENT  "CUDA_VISIBLE_DEVICES=1")
+                 PROPERTY ENVIRONMENT  "CUDA_VISIBLE_DEVICES=0") # Zero based indexed, so only first GPU is visible
```
Comment (agcapps) [inline src/axom/spin/tests/CMakeLists.txt:75]: Boy do I feel silly.  I realized that it's a selector, not a count.  I'd been thinking "OK, we'll have 1 visible device."

**Review by agcapps — APPROVED** [review]
Thank-you, Chris

**Review by kennyweiss — APPROVED** [review]

File: src/axom/spin/tests/CMakeLists.txt (hunk 1 above)
Comment (rhornung67) [reply src/axom/spin/tests/CMakeLists.txt:75]: Thinking in Fortran?!?

**Review by rhornung67 — APPROVED** [review]

File: src/axom/spin/tests/CMakeLists.txt (hunk 1 above)
Comment (white238) [reply src/axom/spin/tests/CMakeLists.txt:75]: copy/paste from the internet!

**Pull request merged**